python/lttoolbox.i.in

```diff
@@ -54,8 +54,9 @@ public:
 
   void lt_proc(int argc, char **argv, char *input_path, char *output_path)
   {
-    FILE* input = fopen(input_path, "r");
-    FILE* output = fopen(output_path, "w");
+	InputFile input;
+	input.open(input_path);
```

Comment:
Inconsistent indentation, which Github weirdly doesn't highlight here like it usually does.